### PR TITLE
add a default value for [-AutoPauseDelayInMinute] parameter of command [New-AzSynapseSparkpool] and [Update-AzSynapseSparkpool]

### PR DESCRIPTION
**Pull request by yanjungao718**
This pr adds a default value for [-AutoPauseDelayInMinute] parameter of command [New-AzSynapseSparkpool] and [Update-AzSynapseSparkpool]
Some customers will specify [EnableAutoPause] without AutoPauseDelayInMinute and will get an error that indicates a non-zero value is needed for that, so add a default value and meanwhile syncing up with Synapse studio side.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Review by wonner — DISMISSED** [review]

File: src/Synapse/Synapse/Common/HelpMessages.cs [hunk 1]
```diff
@@ -88,7 +88,7 @@ public static class HelpMessages
 
         public const string DisableAutoPause = "Indicates whether Auto-pause should be disabled.";
 
-        public const string AutoPauseDelayInMinute = "Number of minutes idle. This parameter must be specified when Auto-pause is enabled.";
+        public const string AutoPauseDelayInMinute = "Number of minutes idle. This parameter can be specified when Auto-pause is enabled. The default value will be [15] if it is not specified manually.";
```
Comment (msJinLei) [inline src/Synapse/Synapse/Common/HelpMessages.cs:91]: You can use a formatted string here.
```suggestion
        public const string AutoPauseDelayInMinute = $"Number of minutes idle. This parameter can be specified when Auto-pause is enabled. The default value will be {SynapseConstants.DefaultAutoPauseDelayInMinute} if it is not specified manually.";
```

**Review by msJinLei — CHANGES_REQUESTED** [review]

**Review by msJinLei — CHANGES_REQUESTED** [review]
Please add description to ChangeLog.md of Synapse

File: src/Synapse/Synapse/Common/HelpMessages.cs (hunk 1 above)
Comment (yanjungao718) [reply src/Synapse/Synapse/Common/HelpMessages.cs]: const interpolated string is unsupporting here, and the help messages must be const strings, so change it to concatenation.

**Review by msJinLei — APPROVED** [review]

**Pull request merged**